### DISCUSSION TITLE
how i use this?

### DISCUSSION THREAD

**Discussion by ooor28** [tylerszabo/RGB-Fusion-Tool#56] [Q&A]
Hi!, i tried to use the set command BUT i dont know what parameters to add can you possibly help?
you can dme me on discord: otiopo#1176

**Comment by tylerszabo** [comment]
There is `--help` option as well as `-h` and `-?` to output the usage information. However, if you want specific example usages check out the test files, as the tests are built from command line examples: [RGBFusionToolExeTests.cs](https://github.com/tylerszabo/RGB-Fusion-Tool/blob/master/RGBFusionTool.Tests/RGBFusionToolExeTests.cs)
